docs/Examples/Flowsheets/index.rst

```diff
@@ -1,8 +1,9 @@
 Flowsheets
 ==========
-This folder contains files that illustrate simulation and optimization of flowsheets.
+This folder contains files that illustrate simulation, optimization and costing of flowsheets.
 
 .. toctree::
     Optimize a hydrodealkylation process with a distillation column <HDA_flowsheet_with_distillation_solution_doc>
+    Apply IDAES Generic Costing Framework for HDA capital cost calculations <HDA_flowsheet_with_costing_doc>
```

Comment:
I think the first conversation we need to have here is what we are going to call the new costing framework - I do not like the name "Generic Costing Framework" as it is simultaneously too vague and carries too many connotations.

Some initial ideas:

* IDAES Process Costing Framework
* IDAES Economics (cribbing blatantly from Aspen, probably too broad for the current state of the tools).

Reply:
I like IDAES Process Costing Framework, particularly in contrast to the IDAES Power Plant Costing Framework (which will have its own example in the near future).